CHANGELOG.md

```diff
@@ -2,10 +2,16 @@
 
 ## Unreleased
 
+## v0.106.1
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.106.1](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.106.1) and the [opentelemetry-collector-contrib v0.106.1](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.106.1) releases where appropriate.
```

Comment:
This actually should be in the `v0.106.0` release notes. I'd put just one bug fix entry here

Reply:
I'll add a bug changelog entry for the dependency update.

In past discussions we've said we want this message in every release. Is there any reason to not include it here? It's in `v0.106.0` as well.

Let me know if I've misunderstood your request here.

Reply:
That's ok with me. I was reading this rather as a changelog entry